firmware/cmake/OsccConfig.cmake

```diff
@@ -0,0 +1,9 @@
+if(KIA_SOUL)
+    add_definitions(-DKIA_SOUL)
+elseif(KIA_SOUL_EV)
+    add_definitions(-DKIA_SOUL_EV)
+elseif(KIA_NIRO)
+        add_definitions(-DKIA_NIRO)
+else()
+    message(FATAL_ERROR "No platform selected")
```

Comment:
Small request - can we change this to indicate that the vehicle platform was selected, and list the possible choices?